.github/workflows/multi-platform.yml

```diff
@@ -104,9 +104,8 @@ jobs:
           context: "{{defaultContext}}"
           push: true
           build-args: |
-            GIT_STRIPPED_VERSION=${{env.GIT_STRIPPED_VERSION}}
-            GIT_VERSION=${{env.GIT_VERSION}}
-            RELEASE_CHANNEL=${{env.RELEASE_CHANNEL}}
+            VERSION=${{env.GIT_VERSION}}
+            GIT_COMMITSHA=${GITHUB_SHA::7}
```

Comment:
Is this the same as other adapter workflows?

Reply:
This entire workflow should be verbatim between all adapters. 

Istio reference - https://github.com/meshery/meshery-istio/blob/7ecbb21b044923fe0ccc38f329bdf94d41446f72/.github/workflows/multi-platform.yml#L107-L109

Reply:
yeah, that is the thing. 
This is why we don't see the version while hovering on Meshery UI. Because we are not baking the correct arguments during image builds.
This is required for https://github.com/meshery/meshery-consul/blob/master/Dockerfile#L2-L3

Reply:
These changes in the multiplatform.yml workflow need to be replicated across all adapters.

Reply:
How is the release channel being accounted for if you're removing it?

Reply:
that is handled by https://github.com/meshery/meshery-istio/blob/7ecbb21b044923fe0ccc38f329bdf94d41446f72/.github/workflows/multi-platform.yml#L82
Docker meta step (docker/metadata-action@v3).
It adds all the necessary metadata to the image. It is doing that through the env variables set in the workflow.

While build args are specifically the ones that are passed as variables to the Dockerfile to be used for specific steps/stages commands. Like this one https://github.com/meshery/meshery-consul/blob/master/Dockerfile#L6

Reply:
Ah, yes... How quickly you forget the things that you have written (referring to myself, in this case).